static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -75,7 +81,26 @@ export function ReleaseSearchBar({
     return acc;
   }, {});
 
-  return (
+  return organization.features.includes('search-query-builder-releases') ? (
+    <SearchQueryBuilder
+      initialQuery={widgetQuery.conditions}
+      filterKeySections={filterKeySections}
+      filterKeys={supportedTags}
+      getTagValues={memoize(
```

Comment:
No need to wrap in `memoize` since the component will deal with caching for you now

Reply:
I thought the default messages were pretty good, but I added explicit ones 👍🏻